### PR TITLE
Example Updates

### PR DESCRIPTION
**Pull request by linuxwolf**
* Regenerated complex example to be correct
* Updated some text based on historical precedent

### CODE REVIEW

File: example/index.js [hunk 1]
```diff
@@ -46,7 +46,7 @@ var samples = {
     "iss": "Upstream CDN Inc",
     "jti": "5DAafLhZAfhsbe",
     "nbf": 1474243200,
-    "sub": "uri-regex:http://cdni.example/foo/bar/baz/[0-9]{3}.png"
+    "sub": "uri-regex:http://cdni\\.example/foo/bar/baz/[0-9]{3}.png"
```
Comment (PSUdaemon) [inline example/index.js]: Don't we need to escape the `.` in `.png` too?

File: draft-ietf-cdni-uri-signing.xml [hunk 2]
```diff
@@ -1305,59 +1319,59 @@ http://*/folder/content-83112371/quality_*/segment????.mp4
...
+{
+  "aud": "eyJhbGciOiJkaXIiLCJraWQiOiJmLVdianhCQzNkUHVJM2QyNGtQMmh
+  mdm9zN1F6Njg4VVRpNmFCMGhOOTk4IiwiZW5jIjoiQTEyOEdDTSJ9..MlXN_ddG
+  DqeZv1Om.siON0_WjzXkzTT-eGLCiQg.f5g8YPO4hsQAxB7SvCc0uw",
+  "exp": 1474243500,
+  "iat": 1474243200,
+  "iss": "Upstream CDN Inc",
+  "jti": "5DAafLhZAfhsbe",
+  "nbf": 1474243200,
+  "sub": "uri-regex:http://cdni\\.example/foo/bar/baz/[0-9]{3}.png"
```
Comment (PSUdaemon) [inline draft-ietf-cdni-uri-signing.xml]: Also, I just want to confirm that we need both the `\`'s? I see when I render it that they are both there, but assuming this is a JSON requirement, right?

Comment (linuxwolf) [inline draft-ietf-cdni-uri-signing.xml]: This is a JSON requirement; '\' is the escape character.  Essentially '\.' is escaping the '.' from the string (which is unnecessary in JSON) but not the regex, so you really need it to be '\\.'

**Comment by PSUdaemon** [comment]
You are a machine, sir. Thank you.

**Pull request merged**